src/history.ts

```diff
@@ -52,37 +52,32 @@ export class NavigationHistory<S extends object>
     return this.#options[State] ?? undefined;
   }
 
-  back(): unknown;
-  back(): void;
-  back(): unknown {
+  back(): void {
     const entries = this.#navigation.entries();
     const index = this.#navigation.currentEntry?.index ?? -1;
     const back = entries[index - 1];
     const url = back?.url;
     if (!url) throw new InvalidStateError("Cannot go back");
-    return this[AppLocationTransitionURL](url, () =>
+    this[AppLocationTransitionURL](url, () =>
```

Comment:
History API always returns `void` afaik

Reply:
These were intentional to support testing efforts rather than the actual exact matching API.